src/components/ButtonGroup/__tests__/buttonGroup-snapshot-tests.jest.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import ButtonGroup from "../ButtonGroup";
+
+const option = [
+  { value: 1, text: "Alpha" },
+  { value: 2, text: "Beta" },
+  { value: 3, text: "Gamma" },
+  { value: 4, text: "Delta" },
+]
+
+describe("ButtonGroup renders correctly", () => {
+    it("with value", () => {
+      const tree = renderer.create(<ButtonGroup value="button" options={option} />).toJSON();
```

Comment:
Please pass a value which is one of the buttons group given values (in this tests the possible values are the numbers between 1-4)